.travis.yml

```diff
@@ -7,6 +7,16 @@ services:
 # a lightweight environment
 language: c
 
+env:
+    - DC=dmd1 DIST=trusty F=production
+    - DC=dmd1 DIST=trusty F=devel
+    - DC=dmd1 DIST=xenial F=production
+    - DC=dmd1 DIST=xenial F=devel
+    - DC=dmd-transitional DIST=trusty F=production
+    - DC=dmd-transitional DIST=trusty F=devel
+    - DC=dmd-transitional DIST=xenial F=production
+    - DC=dmd-transitional DIST=xenial F=devel
```

Comment:
I'm not convinced that approach will be easier and it might be less efficient, and explode the image tags (I guess we'll have to use something like `v1-dmd-1.080-trusty` for example, or do you think we can completely remove the "image version" (v1) in this case, but then how this interact with the base image?). Changing the way we install the compiler seems trivial and adding more compilers once we do that it's just a matter of calling the dmd building script again with a different version.

Since this is not trivial, maybe we should move the discussion to a cachalot issue. Can you create an issue there with a detailed proposal?